### PR TITLE
TCVP-2514 PR 1

### PR DESCRIPTION
**Pull request by saikrishnametpalli-nttdata**
# Description

This PR includes the following proposed change(s):

- TCVP-2514
- Added the change plea

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Improvement
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.html [hunk 1]
```diff
@@ -225,6 +225,65 @@ <h3 style="padding-left: 0.2rem;">JJ decision</h3>
...
+    <div class="row mt-1">
+      <div class="col-md-4">
+        <span class="section-grid-header">Date/Time</span>
+      </div>
+      <div class="col-md-8">
+        <div *ngIf="!isViewOnly && jjDisputedCount">
+          <input type="datetime-local" formControlName="latestPleaUpdateTs" class="section-grid-text latest-plea-text" >
+        </div>
+        <div *ngIf="isViewOnly && jjDisputedCount && jjDisputedCount?.latestPleaUpdateTs">
+          <span class="section-grid-text">{{ jjDisputedCount?.latestPleaUpdateTs | date: "MM/dd/YYYY HH:mm" }}</span>
```
Comment (KDNeufeld) [inline src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.html:257]: Just confirming the date format is this, and not YYYY/MM/dd HH:mm (which is the format on other screens). But it's true that I don't think project has a standard.

Comment (saikrishnametpalli-nttdata) [reply src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.html:257]: I have put this format similar to other input date formats

File: src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts [hunk 2]
```diff
@@ -326,6 +344,22 @@ export class JJCountComponent implements OnInit {
...
+  //Latest Plea
+  bindLatestPlea(value){
+    this.form.controls.latestPlea.setValue(value);
+  }
+
+  bindLatestPleaUpdateTs(value){
+    this.form.controls.latestPleaUpdateTs.setValue(this.toDateString(new Date(value)));
+  }
+
+  toDateString(date: Date): string {
```
Comment (KDNeufeld) [inline src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts:357]: Can we add this to DateUtil?
/src/app/shared/utils/date-util.ts

Comment (Sharkww) [reply src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts:357]: I think we can store it as static variable

Comment (Sharkww) [reply src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts:357]: Please use datepipe to handle it, I have implemented similar coding in other components, you can reference to it

**Review by Sharkww — CHANGES_REQUESTED** [review]

File: src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.html [hunk 3]
```diff
@@ -225,6 +225,65 @@ <h3 style="padding-left: 0.2rem;">JJ decision</h3>
...
+        </div>
+      </div>
+    </div>
+    <div class="row mt-1">
+      <div class="col-md-4">
+        <span class="section-grid-header">Date/Time</span>
+      </div>
+      <div class="col-md-8">
+        <div *ngIf="!isViewOnly && jjDisputedCount">
+          <input type="datetime-local" formControlName="latestPleaUpdateTs" class="section-grid-text latest-plea-text" >
```
[the remark this answers is not in the document]
Comment (saikrishnametpalli-nttdata) [reply src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.html:254]: Made the requested changes

File: src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts [hunk 4]
```diff
@@ -4,6 +4,8 @@ import { JJDispute } from '../../../services/jj-dispute.service';
 import { JJDisputedCount, JJDisputedCountAppearInCourt, JJDisputedCountIncludesSurcharge, JJDisputedCountLatestPlea, JJDisputedCountPlea, JJDisputedCountRequestReduction, JJDisputedCountRequestTimeToPay, JJDisputedCountRoPAbatement, JJDisputedCountRoPDismissed, JJDisputedCountRoPFinding, JJDisputedCountRoPForWantOfProsecution, JJDisputedCountRoPJailIntermittent, JJDisputedCountRoPWithdrawn, JJDisputeHearingType } from 'app/api';
 import { MatLegacyRadioChange as MatRadioChange } from '@angular/material/legacy-radio';
 import { LookupsService, Statute } from 'app/services/lookups.service';
+import { DateUtil } from '@shared/utils/date-util';
```
Comment (Sharkww) [inline src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts:7]: remove unused

File: src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (Sharkww) [reply src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.html:254]: can use ngIf else

File: src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts (hunk 4 above)
Comment (saikrishnametpalli-nttdata) [reply src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts]: Done

File: src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (saikrishnametpalli-nttdata) [reply src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.html:254]: Can you please check once and let me know is it the correct way of doing it

**Pull request merged**